tests/acceptance/request_test.py

```diff
@@ -99,6 +114,7 @@ def test_400_if_required_body_is_missing(test_app):
     ).status_code == 400
 
 
+@pytest.mark.xfail
```

Comment:
I'm not sure what the right way to fix this test is. It looks like calling `request.POST` before `request.json` causes the second property to fail because the body can't be re-interpreted as json after it's read as form encoded. However, accessing the attributes in the opposite order succeeds. Thoughts?